prac_06/guitar.py

```diff
@@ -0,0 +1,43 @@
+"""CP1404
+Initialise the guitar class, fields will include name, year and cost.
+"""
+
+YEAR = 2020  # set the current year as a constant
+
+
+class Guitar:
+
+    def __init__(self, name='', year=0, cost=0.0):
+        """initialise the guitar instance"""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """return string formatting"""
+        return "{self.name} ({self.year}) : ${self.cost:.2f}".format(self=self)
+
+    def get_age(self):
+        """return the age of the guitar"""
+        return YEAR - self.year
```

Comment:
If year is made into a class variable you then reference it with "self.YEAR" . Potentially it should be a different name as it is a class variable and not a constant. (might need to be changed if the year is changed.

Reply:
you are right. I see why it should be a class variable instead of a constant.  will amend it.